scripts/analysis_1/nbconverted/0_clean_metadata.py

```diff
@@ -0,0 +1,153 @@
+
+# coding: utf-8
+
+# # Process metadata
+# 
+# Not all experiment ids in the metadata file have corresponding gene expression data.  This notebook checks each experiment id and returns a clean list of experiment ids that have gene expression data.  This file will be used in ```2_generate_data.ipynb```
```

Comment:
Ah, ok. So this is where 104 `experiments` came from. Make distinction between terms to use when you are referring to `experiments` and simulated `experiments`.

Reply:
Yes this is definitely a source of confusion.  For this next simulation experiment, we want to capture experiment level information in the compendium.  So instead of partitioning our samples into experiments, we are partitioning our experiments into bins.  Where each bin is capturing a source of technical variation (i.e. these 3 experiments came from lab A and these other 3 experiments came from lab B and so lab A and B are our bins).

I will try to clean up the variable names that are being used.